addons/transform/org.openhab.transform.feed.test/FeedTransformationServiceTest.launch

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Maybe you ask, because you are not developing with the Eclipse IDE.

It is used from Eclipse to store a launch configuration (in this case for the tests). It makes the execution of the tests in Eclipse as simple as `Right click -> Run as > FeedTransformationServiceTest.launch `.

To be honest it is not working always flawlessly, as some of the required plugins are taken from the workspace (which of course often differ), but generally I think it is nice to have.